lib/matplotlib/_mathtext.py

```diff
@@ -2719,25 +2723,37 @@ def subsuper(self, s, loc, toks):
             # node757
             x = Hlist([Kern(subkern), sub])
             x.shrink()
+            shift_temp = constants.sub1 * xHeight
             if self.is_dropsub(last_char):
                 shift_down = lc_baseline + constants.subdrop * xHeight
+            elif (s.startswith(r'\left', loc)
+                      and nucleus.depth > shift_temp - capHeight / 10):
+                shift_down = nucleus.depth + capHeight / 10
             else:
                 shift_down = constants.sub1 * xHeight
```

Comment:
I think what this is saying is?
```suggestion
            else:
                shift_down = constants.sub1 * xHeight
                if s.startswith(r'\left', loc):
                    shift_down = max(shift_down, nucleus.depth + capHeight / 10)
```
and similar for the others.

Reply:
Ah yes, it is. This is a lot clearer, too. Will add a commit with these changes.